ui/src/app/applications/components/application-urls.tsx

```diff
@@ -3,7 +3,7 @@ import * as React from 'react';
 
 export const ApplicationURLs = ({urls}: {urls: string[]}) => {
     return (
-        ((urls || []).length > 0 && (
+        ((urls || []).sort().length > 0 && (
```

Comment:
Thank you @bjoernhaeuser . 
I think the implemented code works fine because the method `sort` sorts array in-place. Just to make it easier to read code can you move sort to line 4? e.g.

```
export const ApplicationURLs = ({urls}: {urls: string[]}) => {
    (urls || []).sort();
    return (
        ((urls || []).length > 0 && (
        ((urls || []).length > 0 && (
```

Reply:
Resolved with 3f4c246691cabaf388a1420bae4aae1d5b9ec9e4